addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -1044,8 +1044,7 @@ var StatementModel = BasicModel.extend({
      * @returns {object}
      */
     _formatToProcessReconciliation: function (line, prop) {
-        // Do not forward port in master. @CSN will change this
```

Comment:
@csnauwaert is it safe to revert it now in master?
I would leave it until you fix it correctly.

Reply:
@KangOl The branch with the fix is not yet finished and this is not fixed yet in master, you can forward port it, I'll revert/fix it within my branch